packages/keystone/lib/adapters/index.js

```diff
@@ -113,7 +113,7 @@ class BaseListAdapter {
   }
 
   async delete(id) {
-    return this.onPostRead(this._delete(id));
+    return this._delete(id);
```

Comment:
Given the differences between adapters, this is a good opportunity to normalise the possible return value to be just the `id`:

```suggestion
   await this._delete(id);
   return id;
```

Reply:
Ah, was just typing something similar vv.

But yes, or this ^^

Reply:
Or not return anything..